src/codegen/general.rs

```diff
@@ -134,14 +134,26 @@ pub fn define_boxed_type(
     ));
     try!(writeln!(w, ""));
     try!(writeln!(w, "\tmatch fn {{"));
-    try!(writeln!(
-        w,
-        "\t\tcopy => |ptr| ffi::{}(mut_override(ptr)),",
-        copy_fn
-    ));
-    try!(writeln!(w, "\t\tfree => |ptr| ffi::{}(ptr),", free_fn));
-    if let Some(ref get_type_fn) = *get_type_fn {
+    if let Some(ref get_type_fn) = get_type_fn {
```

Comment:
Clippy nags about this:
```
warning: this pattern creates a reference to a reference
   --> src\codegen\general.rs:137:17
    |
137 |     if let Some(ref get_type_fn) = get_type_fn {
    |                 ^^^^^^^^^^^^^^^ help: change this to: `get_type_fn`
    |
    = note: #[warn(needless_borrow)] on by default
    = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.186/index.html#needless_borrow
```
same in other 2 here

Reply:
How do you run clippy btw?

Reply:
Through `cargo clippy`.
`cargo install -f clippy` had sometimes build error, but with my current nightly it works.

Reply:
You need nightly in any case. If you use rustup, install as second toolchain, then cargo +nightly install clippy and cargo +nightly clippy for running it.

Stable can stay your default toolchain